playwright/_impl/_connection.py

```diff
@@ -319,11 +319,12 @@ def _replace_guids_with_channels(self, payload: Any) -> Any:
         return payload
 
 
-def from_channel(channel: Channel) -> Any:
+def from_channel(channel: Channel) -> ChannelOwner:
```

Comment:
As all inherit from ChannelOwner we eliminate runtime casts

Reply:
You don't need this change to move to cast. It does not hurt, but this comment is misleading

Reply:
Updated comment